lib/stdlib/src/maps.erl

```diff
@@ -231,14 +215,11 @@ filter_1(Pred, Iter) ->
     end.
 
 -spec fold(Fun,Init,MapOrIter) -> Acc when
-    Fun :: fun((K, V, AccIn) -> AccOut),
+    Fun :: fun((Key, Value, AccIn) -> AccOut),
```

Comment:
The documentation references `K` and `V` through `anno` tags, which need to be updated with the new names.